website/docs/docs/build/snapshots.md

```diff
@@ -58,6 +60,35 @@ select * from {{ source('jaffle_shop', 'orders') }}
 
 </File>
 
+</VersionBlock>
+
+<VersionBlock firstVersion="1.9">
+
+<File name='snapshots/orders_snapshot.sql'>
+
+```sql
+{% snapshot orders_snapshot %}
+
+{{
+    config(
+      unique_key='id',
+      schema='snapshots', // optional config. If not defined, the snapshot will use the `generate_schema_name` macro.
```

Comment:
`schema` and `database` are the new configs that _use_ the `generate_schema_name` / `generate_database_name` macros like other dbt resources

`target_schema` and `target_database` are the old configs